StabilityClient.Net/StabilityClient.Net/RequestBuilder.cs

```diff
@@ -1,9 +1,10 @@
+using Google.Protobuf;
 using Gooseai;
 
 namespace StabilityClient.Net;
 
 public class RequestBuilder {
-    public const string DefaultEngineId = "stable-diffusion-512-v2-1";
+    public const string DefaultEngineId = "stable-diffusion-v1-5";
```

Comment:
There are some warning that I can see in review. Please take a look and try to resolve them. 
![image](https://user-images.githubusercontent.com/19012171/217600918-004220b9-3bb9-4540-84a8-31dcfcffb1d4.png)

Reply:
I see that added comments helped for this issue. However, there are still some warning in different files.